src/shared/components/input/RadioButton.tsx

```diff
@@ -121,20 +26,26 @@ export const RadioButton: FC<RadioButtonProps> = ({
     const [isFocusWithin, setIsFocusWithin] = useState(false);
 
     return (
-        <Label htmlFor={id} disabled={disabled}>
-            <FlexContainer flexDirection='column' alignItems='flex-start' gridGap='5px'>
+        <label
+            className={cn('relative flex min-h-5 w-5', {
+                'cursor-not-allowed pointer-events-none': disabled,
+                'cursor-pointer pointer-events-auto group': !disabled,
+            })}
+            htmlFor={id}
+        >
+            <div className='flex flex-col items-start gap-[0.3125rem]'>
                 {title && (
-                    <Paragraph $typeset='headline' color='base'>
+                    <p className='text-base font-normal leading-tight text-subtle-black dark:text-white'>
```

Comment:
might be worth it to add default text color to the `body` tag instead so it always has the regular light + dark mode texts applied and we only overwrite it where needed. That way you don't have to define this for each `p` tag, but have a look if it breaks anything in the UI as it might get too involved for this PR if it's messing up other areas